atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -2113,10 +2125,13 @@ public void compactInternally(TableReference tableRef) {
             alterGcAndTombstone(keyspace, tableRef, 0, 0.0f);
             compactionManager.get().performTombstoneCompaction(timeoutInSeconds, keyspace, tableRef);
         } catch (TimeoutException e) {
-            log.error("Compaction for {}.{} could not finish in {} seconds.", keyspace, tableRef, timeoutInSeconds, e);
-            log.error("Compaction status: {}", compactionManager.get().getCompactionStatus());
+            log.error("Compaction for {}.{} could not finish in {} seconds.", UnsafeArg.of("keyspace", keyspace),
+                    UnsafeArg.of("table", tableRef), SafeArg.of("timeout", timeoutInSeconds), e);
+            log.error("Compaction status: {}",
+                    UnsafeArg.of("compactionStatus", compactionManager.get().getCompactionStatus()));
```

Comment:
Thought this could be safe, but good chance it will leak table names so let's stick with unsafe.